benchmarks/votable.py

```diff
@@ -20,21 +17,33 @@
 id_data = np.arange(LARGE_SIZE, dtype=np.int64)
 flag_data = np.random.choice([True, False], LARGE_SIZE)
 quality_data = np.random.randint(0, 256, LARGE_SIZE, dtype=np.uint8)
+bool_data = np.random.randint(0, 2, LARGE_SIZE).astype(bool)
 
 short_names = np.array([f"OBJ_{i:08d}" for i in range(LARGE_SIZE)])
-filter_names = np.random.choice(['u', 'g', 'r', 'i', 'z', 'Y'], LARGE_SIZE)
+filter_names = np.random.choice(["u", "g", "r", "i", "z", "Y"], LARGE_SIZE)
```

Comment:
Why is the pull request cluttered with unrelated formatting changes? I am only highlighting it here, but it is a problem throughout.

Reply:
I've reverted those changes, I can put in a separate PR to address the PEP8 issues.